experimental/nodejs-functions/templates/simple/.vscode/launch.json

```diff
@@ -0,0 +1,17 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Appsody: Attach node debugger",
+            "type": "node",
+            "request": "attach",
+            "port": 9229,
+            "remoteRoot": "/project/user-app",
```

Comment:
the remote root should be: `/project/user-app/functions` ?

Reply:
Good spot! Updated.